R/frames.R

```diff
@@ -65,9 +70,36 @@ frame_1<- function(map,
 
 
   server <- function(input, output) {
-    output$map <- renderLeaflet({
-      map
-    })
+    output$img <- renderImage({
+
+      # create a tempdir
+      myDir = tempdir()
+
+      # save map as png file
+      mapshot(map,file = paste0(myDir,"map.png"))
+      mymap <- image_read(paste0(myDir,"map.png"))
+      img_info_mymap <- image_info(mymap)
+
+      # load mask and resize
+      mask <- image_read('masks/circular_mask.png')

```

Comment:
Since its possible to switch out the masks, it would be good to just have an argument `mask` in the function and have a few named arguments e.g. c("circle","square", "house") and do some `if else` logic to parse through it. 

It might also be nice to make a separate function that would allow the user to provide a custom mask as well.